virttest/utils_net.py

```diff
@@ -4351,3 +4351,46 @@ def delete_ovs_bridge(ovs_bridge_name, session=None, ignore_status=False):
     return utils_misc.cmd_status_output(tmux_cmd, shell=True, verbose=True,
                                         ignore_status=ignore_status,
                                         session=session)
+
+
+def create_linux_bridge(linux_bridge_name, iface_name, ignore_status=False):
+    """
+    Create linux bridge and connect a physical interface to the bridge via tmux command on local host
+    1. use tmux to avoid the network broken;
+    2. add a physical interface to the bridge, and set the bridge up;
+
+    :param linux_bridge_name: The linux bridge
+    :param iface_name: the physical interface to be attached to the bridge
+    :param ignore_status: Whether to raise an exception when command fails
+    :return: bridge created or raise exception
+    """
+    # Create bridge
+    if not utils_package.package_install(['tmux', 'dhcp-client']):
+        raise exceptions.TestError("Failed to install the required packages.")
+    current_iface = process.run("ls /sys/class/net", shell=True, verbose=True, ignore_status=True).stdout_text.split()
```

Comment:
ignore_status=True or should be ignore_status=False ?

Reply:
have updated to:
"br_path = "/sys/class/net/%s" % linux_bridge_name
if os.path.exists(br_path):
    raise exceptions.TestError(xxx)"
So the process.run() is not needed now.